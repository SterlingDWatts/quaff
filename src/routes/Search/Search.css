.Search {
  padding-top: 44px;
  height: 100%;
}
