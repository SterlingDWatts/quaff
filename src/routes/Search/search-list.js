export default [
  {
    term: "Photosynthesis",
  },
  {
    term: "Alcohol",
  },
  {
    term: "Yeast",
  },
  {
    term: "Soil",
  },
  {
    term: "Clay",
  },
  {
    term: "Sugar",
  },
  {
    term: "Acid",
  },
  {
    term: "Tannin",
  },
  {
    term: "Body",
  },
  {
    term: "Diurnal shift",
  },
  {
    term: "Climate",
  },
  {
    term: "Weather",
  },
  {
    term: "Water",
  },
  {
    term: "Sunlight",
  },
  {
    term: "Warmth",
  },
  {
    term: "Carbon dioxide",
  },
  {
    term: "Nutrient",
  },
  {
    term: "Glucose",
  },
  {
    term: "Vine",
  },
  {
    term: "Grape",
  },
  {
    term: "Wine",
  },
  {
    term: "Vintage",
  },
  {
    term: "Ripe",
  },
  {
    term: "Mediterranean Climate",
  },
  {
    term: "Continental Climate",
  },
  {
    term: "Maritime Climate",
  },
  {
    term: "Cool Climate",
  },
  {
    term: "Warm Climate",
  },
  {
    term: "Moderate Climate",
  },
  {
    term: "Hot Climate",
  },
  {
    term: "Champagne",
  },
  {
    term: "Bordeaux",
  },
  {
    term: "Rain",
  },
  {
    term: "Oxygen",
  },
  {
    term: "Fruit set",
  },
  {
    term: "Flowering",
  },
  {
    term: "Harvest",
  },
  {
    term: "Fog",
  },
  {
    term: "Hill",
  },
  {
    term: "Macroclimate",
  },
  {
    term: "Mesoclimate",
  },
  {
    term: "Microclimate",
  },
  {
    term: "AVA",
  },
  {
    term: "Appellation",
  },
  {
    term: "Temperature",
  },
  {
    term: "Wind",
  },
  {
    term: "Ocean currents",
  },
  {
    term: "Canopy",
  },
  {
    term: "Row",
  },
  {
    term: "Shade",
  },
  {
    term: "Slope",
  },
  {
    term: "Pulp",
  },
  {
    term: "Pip",
  },
  {
    term: "Seed",
  },
  {
    term: "Fermentation",
  },
  {
    term: "Crisp",
  },
  {
    term: "Dry",
  },
  {
    term: "Refreshing",
  },
  {
    term: "Skin",
  },
  {
    term: "Color",
  },
  {
    term: "Flavor",
  },
  {
    term: "Bloom",
  },
  {
    term: "Bacteria",
  },
  {
    term: "Berry",
  },
  {
    term: "Variety",
  },
  {
    term: "Location",
  },
  {
    term: "Bud",
  },
  {
    term: "Bud break",
  },
  {
    term: "Spring",
  },
  {
    term: "Frost",
  },
  {
    term: "Pinot Noir",
  },
  {
    term: "Structure",
  },
  {
    term: "Aging",
  },
  {
    term: "Vitis Vinifera",
  },
  {
    term: "Species",
  },
  {
    term: "Cultivar",
  },
  {
    term: "Phyloxera",
  },
  {
    term: "Root",
  },
  {
    term: "Louse",
  },
  {
    term: "Pest",
  },
  {
    term: "Disease",
  },
  {
    term: "Graft",
  },
  {
    term: "Trunk",
  },
  {
    term: "Rootstock",
  },
  {
    term: "Clone",
  },
];
