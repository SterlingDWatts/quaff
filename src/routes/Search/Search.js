import React, { Component } from "react";
import { Link } from "react-router-dom";
import SubNavBar from "../../components/SubNavBar/SubNavBar";
import { ExploreSquare } from "../../components/Utils/Utils";
import "./Search.css";

class Search extends Component {
  render() {
    return (
      <div className="Search">
        <SubNavBar />
        <ExploreSquare></ExploreSquare>
      </div>
    );
  }
}

export default Search;
