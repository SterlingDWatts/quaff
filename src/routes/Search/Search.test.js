import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import Search from "./Search";

describe("Search Component", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Search />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Search />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
