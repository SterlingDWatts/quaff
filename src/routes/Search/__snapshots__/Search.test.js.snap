// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Component renders the UI as expected 1`] = `
<div
  className="Search"
>
  <nav
    className="SubNavBar"
  >
    <a
      href="/learn"
      onClick={[Function]}
    >
      <div
        className="SubNavBarPill"
      >
        Learn
      </div>
    </a>
    <a
      href="/study"
      onClick={[Function]}
    >
      <div
        className="SubNavBarPill"
      >
        Study
      </div>
    </a>
    <a
      href="/search"
      onClick={[Function]}
    >
      <div
        className="SubNavBarPill"
      >
        Search
      </div>
    </a>
  </nav>
  <div
    className="ExploreSquare "
  >
    <div
      className="ExploreSquare__border"
    >
      <div
        className="ExploreSquare__container"
      />
    </div>
  </div>
</div>
`;
